Fix for module not found issue

Hi,

Yesterday we ran into a ```Module not found: 'http-parser-js'``` error with this library.

I managed to replicate the issue on Linux (the original error was happening on Heroku), and discovered that ```npm install``` was not installing the http-parser.js file. I then tried to install previous versions of the library to isolate the cause of this, and found that this commit was where that issue was occurring:

https://github.com/creationix/http-parser-js/commit/0bd74fde0627e0e549dfe341ddfe06f521515ee0

It looks like on older versions of npm, the files property in the package.json was an empty array, so only the package.json and README.md files were being installed, and not the http-parser.js (even though it is referenced in the main property).

I added the reference in the files property, and the issue was fixed.

--- a/package.json
+++ b/package.json
@@ -10,7 +10,9 @@
     "type": "git",
     "url": "git://github.com/creationix/http-parser-js.git"
   },
-  "files": [],
+  "files": [
+    "http-parser.js"
+  ],
   "keywords": [
     "http"
   ],

